### ISSUE TITLE
Deprecate and remove "dashboards" code

### ISSUE THREAD

**Issue by jarednorman** [solidusio/solidus#4860]
None of this provides any real value. Adding a custom dashboard is trivial and I really don't think we need a custom permission set and empty controller/view. 

- https://github.com/solidusio/solidus/blob/master/backend/app/controllers/spree/admin/dashboards_controller.rb
- https://github.com/solidusio/solidus/blob/master/backend/app/views/spree/admin/dashboards/home.html.erb
- https://github.com/solidusio/solidus/blob/master/core/lib/spree/permission_sets/dashboard_display.rb
- https://github.com/solidusio/solidus/blob/master/core/spec/models/spree/permission_sets/dashboard_display_spec.rb

**Comment by jarednorman** [comment]
[This](https://github.com/solidusio/solidus/pull/167/files#diff-3d5717d890ced55ac628baf2e2bda377427a1c00a90939121da7d3d90b48020eR13-R17) could be replaced with a default admin location.

**Comment by forkata** [comment]
I doubt anyone is using this, but I guess we would have to deprecate and then remove this in a major version?

**Comment by jarednorman** [comment]
Yeah, that's my thought. It was introduced alongside the permission sets, but I've never seen a store that used it.

**Comment by lauriejefferson** [comment]
I would like to work on this issue.  Is it still open?

**Comment by jarednorman** [comment]
The new admin renders this kind of moot, but it couldn't hurt to still do this.

**Comment by lauriejefferson** [comment]
@jarednorman Do you want to add deprecation warnings in the files listed above or just remove them without the warnings?

**Comment by jarednorman** [comment]
I think they should be deprecated first.
